src/components/settings.rs

```diff
@@ -5,7 +5,7 @@ use dioxus::prelude::*;
 #[cfg(debug_assertions)]
 use dioxus::logger::tracing::trace;
 
-use crate::{ESPLORA_ENDPOINT, NETWORK};
+use crate::{ESPLORA_ENDPOINT, NETWORK, validation::*};
```

Comment:
no globbing import the stuff explicitly, this is not Python...
We only glob preludes like `dioxus::prelude::*` or in tests with `use super::*`